public/app/core/components/TimePicker/TimePickerWithHistory.tsx

```diff
@@ -34,6 +35,12 @@ export const TimePickerWithHistory = (props: Props) => {
               onAppendToHistory(value, values, onSaveToStore);
               props.onChange(value);
             }}
+            onPasteError={(clipboardContent: string) =>
```

Comment:
when it comes to buttons, I'm only showing a toast when there's an error on paste. however, when using keyboard shortcuts, I show a success toast on copy, so I'm wondering if we should add the same logic when copying successfully on the button click.